### PR TITLE
feat(CreateCommunityPopup): Add Community Banner to Create Community Modal

### LINKED ISSUE #6726: Add Community Banner to Create Community Modal
### Description

The Create Community Modal is missing the Community Banner. This already exists in Edit Community, so it should just be a matter of adding this to the create community modal as well.

Screen Shot 2022-07-29 at 3 54 23 PM

### PR DESCRIPTION
**Pull request by caybro**
as status-go already supports parsing the banner JSON string, just pass
it along the NIM backend

Closes #6726

### What does the PR do

Implements support for adding community banner on its creation

### Affected areas

CreateCommunityPopup

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

Create community:
Snímek obrazovky z 2022-08-17 14-26-51

Edit community (for verification):
Snímek obrazovky z 2022-08-17 14-27-15

### CODE REVIEW

File: ui/nim-status-client.pro [hunk 1]
```diff
@@ -23,6 +23,7 @@ TRANSLATIONS += \
 
 OTHER_FILES += $$files("$$PWD/*.qml", true)
 OTHER_FILES += $$files("$$PWD/*.js", true)
+OTHER_FILES += $$files("$$PWD/../src/*.nim", true)
```
Comment (osmaczko) [inline ui/nim-status-client.pro:26]: Searching files in qt creator (ctrl+k) is already very slow (at least for me), with all `*.nim` files it will probably be even slower :disappointed: But I understand it's your IDE of choice for nim development and can't forbid it here :man_shrugging:

**Review by osmaczko — APPROVED** [review]
LGTM

File: ui/nim-status-client.pro (hunk 1 above)
Comment (caybro) [reply ui/nim-status-client.pro:26]: What's your QtC version? Here in 8.0.1 it's super fast

Comment (PascalPrecht) [reply ui/nim-status-client.pro:26]: It's unclear why this change is needed for this commit

Comment (osmaczko) [reply ui/nim-status-client.pro:26]: Was 7.0.x, just have updated to 8.0.1, it's super fast! :fireworks:

Comment (caybro) [reply ui/nim-status-client.pro:26]: Yeah it is, also the cmake/C++ support is so much better

Comment (micieslak) [inline ui/nim-status-client.pro:26]: is it intentional? If we want to add it to the config it should go to a separate pr imo :)

**Review by micieslak — APPROVED** [review]
LGTM, but please check out my comment
